docs/understanding-tracking-design/managing-tracking-scenarios-console/index.md

```diff
@@ -0,0 +1,182 @@
+
+Similar to the **Add implementation rules** modal, you will be able to add multiple entities from the list of entites displayed, that can be sourced from your [Iglu](/docs/understanding-your-pipeline/glossary-of-terms/index.md#iglu) or custom user defined entites. For each of the entities, you can view detailed information about the properties of the entity by clicking **'View properties'**. A search bar will also be available to help search and filter by entity name.
+
+Once you've selected the entities you want to add to the tracking scenario, click the **'Select entities'** button to confirm or the **'Cancel'** button to discard your selections.
+
+Once selected you will be presented with a new view inside the modal where you can select the cardinality of the entities.
+
+![](images/ts-11.png)
+
+It will save the entites on **'Save entities'** click or back to the previous selection view clicking **'Back'**.
+
+Once the entities are saved, you will see a list of entities in the tracking detail view where you can edit, add or remove them.
+
+![](images/ts-12.png)
+
+* * *
+
+## Publishing a Tracking Scenario
+
+When you've configured a tracking scenario and you are ready to publish it, go to the **Tracking scenario** tab view, select the scenario by clicking on its name, and then click the **'Publish scenario'** button at the right side of the scenario name from the detailed view.
+
+![](images/ts-13.png)
+
+Once the scenario is published, it will no longer appear with the status **Draft** but instead with the status **Published** in the list view. When a scenario is published, you won't be able to edit it anymore unless you create a new **Draft** version. To do this  click on the scenario name from the list of tracking scenarios, and in the detailed view, instead of the previous **'Publish scenario'** button, you will see a **'Create a draft & edit'** button.
+
+![](images/ts-14.png)
+
+* * *
+
+## Scharing a Tracking Scenario
+
+You may want to share a tracking scenario, whether it is published or still a draft. To do this from the **Tracking scenario** tab, click on the tracking scenario name you want to share, and then from the detailed view, click the **'Share scenario'** button near to the publish button.
+
+![](images/ts-15.png)
+
+Then a modal view with a link with a **'Copy'** button will appear.
+
+![](images/ts-16.png)
+
+* * *
+
+## Deleting Drafted Tracking Scenarios
+
+You can delete a tracking scenario, but this is only possible for *Draft* versions. To delete a Draft tracking scenario, got to the **Tracking scenarios** tab and click the **'Delete'** button.
+
+A confirmation modal will appear.
+
+![](images/ts-17.png)
+
+:::warning
+This opeartion can't be reverted.
+:::
+
+## Deprecating Published Tracking Scenarios
+
+You can deprecate a tracking scenario, but this is only possible for *Published* versions. To deprecate a Published tracking scenario, go to the **Tracking scenarios** tab and click the **'Deprecate'** button.
```

Comment:
Could you add a sentence about what it means for a scenario to be deprecated? Does anything special happen to the new tracked events, for instance?

Reply:
I added the sentence above:

> Deprecating a tracking scenario is a signal to developers that this scenario is no longer in use and should not be tracked anymore. However, it's important to note that deprecating a tracking scenario does not automatically prevent new events from being tracked under this scenario. Developers will need to manually stop tracking events for the deprecated scenario.

Let me know if it makes sense to you.